docs/DeveloperGuide.adoc

```diff
@@ -727,11 +727,69 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 
 |`* * *` |user |delete a person |remove entries that I no longer need
 
+|`* * *` |user |clear my address book |refresh it quickly
+
+|`* * *` |user with many friends |list all friends with a certain tag |group my friend by tag easily
+
+|`* * *` |As a user |add a person with link:#blank-information[blank information] |add him even if I do not know some details (e.g his address)
+
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |user |get help information |know where is wrong whe I get errors
+
+|`* * *` |user |update contact's information |correct or add their information
+
+|`* * *` |user |list contacts |view who are my contacts
+
+|`* * *` |user |hide private details  |minimise the chance of someone else seeing them by accident
+
+|`* * *` |user |undo decision  |revert changes in case of mistake
+
+|`* * *` |lazy user |have auto complete when I type  |type less word
+
+|`* *` |user who share my device with others| password to lock my address book| prevent others looking at it
```

Comment:
Take note of formatting. Some different number of spaces before and after `|`.

Found in several of your user stories.